test/index.test.js

```diff
@@ -59,56 +57,66 @@ describe('Ramen', function() {
   });
 
   describe('loading', function() {
-    it('should load', function(done){
+    it('should load', function(done) {
       analytics.load(ramen, done);
     });
   });
 
   describe('after loading', function() {
-    beforeEach(function(done){
+    beforeEach(function(done) {
       analytics.once('ready', done);
       analytics.initialize();
     });
 
-    describe('#group', function() {
+    describe('#page', function() {
       beforeEach(function() {
         analytics.stub(window.Ramen, 'go');
       });
 
-      it('should not call Ramen.go if ramenSettings is blank', function() {
-        analytics.group('id');
-        analytics.assert(!window.ramenSettings);
+      it('should not call Ramen.go', function() {
+        analytics.page();
         analytics.didNotCall(window.Ramen.go);
       });
 
-      it('should set company ID & call Ramen.go() if ramenSettings exists', function() {
-        window.ramenSettings = {
-          organization_id: '1234567890',
-          user: { email: 'ryan@ramen.is', name: 'Ryan', id: '1234' }
-        };
+      it('should call Ramen.go twice', function() {
+        analytics.identify('12345', { email: 'ryan@ramen.is' });
+        analytics.page();
+        analytics.calledTwice(window.Ramen.go);
```

Comment:
Ah ok cool breakthrough. It's working now. I'm finishing up some refactoring and will have an update in under an hour.